index.html

```diff
@@ -29,38 +29,35 @@
   <meta property="twitter:description" content="Mapping a variety of local 350 groups, events and climate actions around the world.">
   <meta property="twitter:image" content="http://350.org/sites/all/themes/threefifty/logo.gif">
 
-  <link rel="stylesheet" href="css/leaflet.css" />
+  <link rel="stylesheet" href="bower_components/leaflet/dist/leaflet.css" />
   <!--[if lte IE 8]><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.ie.css" /><![endif]-->
   <!--[if IE]><style type="text/css">.leaflet-top { top: 60px; }</style><![endif]-->
   <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.js"></script>

```

Comment:
Good question...I think we can change this particular instance, but in some of the other `index.html` files (like `living-dangerously`, `submit`, and `events`), the CDN link points to a different version (0.6.3). Do you know if that's intentional and if those should be kept at 0.6.3 instead of 0.7.7?